demo/configuration_example.py

```diff
@@ -1,4 +1,14 @@
 # Configuration
 API_URL = "https://localhost:8080/api/v1"
+USE_BASIC_AUTH = False
+SECRET_KEY = "secret_key"
```

Comment:
You aren't using this anywhere. Maybe we should set it up like how it was used here? https://github.com/osu-mist/oauth2-consent-prototype#configuration